src/test/java/com/exasol/validation/GitRepositoryValidatorTest.java

```diff
@@ -58,7 +58,18 @@ void testValidateChangesInvalidDate() {
         when(changesMock.getReleaseDate()).thenReturn(Optional.of(LocalDate.of(2020, 8, 1)));
         when(changesMock.getBody()).thenReturn(Optional.of("## Features"));
         when(changesMock.getFileName()).thenReturn("file");
-        assertThrows(IllegalStateException.class, () -> this.validator.validateChanges(changesMock, "2.1.0"));
+        assertThrows(IllegalStateException.class, () -> this.validator.validateChanges(changesMock, "2.1.0", true));
```

Comment:
Maybe also validate the exception message?

Reply:
Added